src/components/title-with-description/TitleWithDescription.js

```diff
@@ -1,10 +1,10 @@
 import { Box, Typography } from '@mui/material'
 import { styles } from '~/components/title-with-description/title-with-description.styles'
 
-const TitleWithDescription = ({ title, titleStyles, description, descriptionStyles, componentStyles }) => {
+const TitleWithDescription = ({ title, titleStyles, description, descriptionStyles, componentStyles = styles }) => {
   return (
-    <Box sx={ { ...styles.wrapper, ...componentStyles } }>
-      <Typography sx={ { ...styles.title, ...titleStyles } }>
+    <Box sx={ { ...styles.wrapper, ...componentStyles.wrapper } }>
```

Comment:
seems like you set the same styles to this box

Reply:
fixed